### DISCUSSION TITLE
BarDropdown icon

### DISCUSSION THREAD

**Discussion by angelru** [Megabit/Blazorise#2174] [Q&A] [answered]
I want to get a layout like this, but how can I hide the arrow icon, css?
![Captura](https://user-images.githubusercontent.com/11645585/113876343-efa04700-97b7-11eb-9dec-ac5eb37744d8.PNG)

**Comment by stsrki** [comment, marked as answer]
Hmm, I don't see any arrow icon in the image.

**Reply by angelru** [reply]
![Captura](https://user-images.githubusercontent.com/11645585/113877981-7bff3980-97b9-11eb-8f93-b6d973a6bb52.PNG)

**Reply by stsrki** [reply]
In the first image. Is that some custom template or did you customized CSS yourself?

Arrows should be natively supported by the Bootstrap framework.

**Reply by angelru** [reply]
Maybe I explained myself wrong, I want to get a design like in the first image, is it possible with BarDropdown?

**Reply by stsrki** [reply]
```css
.dropdown-toggle::after {
    content: none;
}
```
